src/rebuild.ts

```diff
@@ -19,6 +19,13 @@ export interface RebuildOptions {
    * The version of Electron to build against.
    */
   electronVersion: string;
+  /**
+   * Override the target platform to something other than the host system platform.
+   * Note: This only applies to downloading prebuilt binaries. **It is not possible to cross-compile native modules**
```

Comment:
```suggestion
   * Note: This only applies to downloading prebuilt binaries. **It is not possible to cross-compile native modules.**
```

Tiny punctuation nit but otherwise LGTM